### PR TITLE
Changes in the pattern of Uncategorized Exception Error

### PR DESCRIPTION
**Pull request by kedarbcs16**
From 

UNCATEGORIZED: 1 count(1)
/root/drillAutomation/master/framework/resources/Advanced/runtimefilter/tpch_sf100/parquet/hash_join_date.sql

UNCATEGORIZED: 2 count(1)
/root/drillAutomation/master/framework/resources/Advanced/runtimefilter/tpch_sf100/parquet/broadcast_join_date.sql

To

CATEGORY - UNCATEGORIZED_ERROR count(2)
/root/drillAutomation/master/framework/resources/Advanced/runtimefilter/tpch_sf100/parquet/hash_join_date.sql
/root/drillAutomation/master/framework/resources/Advanced/runtimefilter/tpch_sf100/parquet/broadcast_join_date.sql

### CODE REVIEW

File: framework/src/main/java/org/apache/drill/test/framework/TestDriver.java [hunk 1]
```diff
@@ -448,16 +452,17 @@ else if(executionFailureExceptions.get(index)==null){
             else if(executionFailureExceptions.get(ii)!=null && executionFailureExceptions.get(ii).size()!=0){
               if(ii<DrillTestDefaults.DRILL_EXCEPTION.VALIDATION_ERROR_INVALID_SCHEMA.values().length){
                 if(ii==0)
-                  LOG.info("CATEGORY - "+DrillTestDefaults.DRILL_EXCEPTION.VALIDATION_ERROR_INVALID_SCHEMA.values()[ii]+" count("+executionFailureExceptions.get(ii).size()+")");
+                  LOG.info("CATEGORY - "+DrillTestDefaults.DRILL_EXCEPTION.VALIDATION_ERROR_INVALID_SCHEMA.values()[ii]+"("+executionFailureExceptions.get(ii).size()+")");
```
Comment (Agirish) [inline framework/src/main/java/org/apache/drill/test/framework/TestDriver.java]:
```suggestion
                  LOG.info("CATEGORY - "+DrillTestDefaults.DRILL_EXCEPTION.VALIDATION_ERROR_INVALID_SCHEMA.values()[ii]+" ("+executionFailureExceptions.get(ii).size()+")");
```

File: framework/src/main/java/org/apache/drill/test/framework/TestDriver.java [hunk 2]
```diff
@@ -448,16 +452,17 @@ else if(executionFailureExceptions.get(index)==null){
...
+                else if(ii<executionFailureExceptions.size()-1)
+                  LOG.info("\nCATEGORY - "+DrillTestDefaults.DRILL_EXCEPTION.VALIDATION_ERROR_INVALID_SCHEMA.values()[ii]+"("+executionFailureExceptions.get(ii).size()+")");
```
Comment (Agirish) [inline framework/src/main/java/org/apache/drill/test/framework/TestDriver.java]:
```suggestion
                  LOG.info("\nCATEGORY - "+DrillTestDefaults.DRILL_EXCEPTION.VALIDATION_ERROR_INVALID_SCHEMA.values()[ii]+" ("+executionFailureExceptions.get(ii).size()+")");
```

File: framework/src/main/java/org/apache/drill/test/framework/TestDriver.java [hunk 3]
```diff
@@ -448,16 +452,17 @@ else if(executionFailureExceptions.get(index)==null){
...
                 else
-                  LOG.info("\nCATEGORY - "+DrillTestDefaults.DRILL_EXCEPTION.VALIDATION_ERROR_INVALID_SCHEMA.values()[ii]+" count("+executionFailureExceptions.get(ii).size()+")");
-                  
+                  LOG.info("\n"+DrillTestDefaults.DRILL_EXCEPTION.VALIDATION_ERROR_INVALID_SCHEMA.values()[ii]+": ("+executionFailureExceptions.get(ii).size()+")");  
               }
               else{ 
                 if(ii==0)
-                  LOG.info("UNCATEGORIZED: "+(ii-DrillTestDefaults.DRILL_EXCEPTION.VALIDATION_ERROR_INVALID_SCHEMA.values().length+1)+" count("+executionFailureExceptions.get(ii).size()+")");
+                  LOG.info("UNCATEGORIZED: "+(ii-DrillTestDefaults.DRILL_EXCEPTION.VALIDATION_ERROR_INVALID_SCHEMA.values().length+1)+"("+executionFailureExceptions.get(ii).size()+")");
```
Comment (Agirish) [inline framework/src/main/java/org/apache/drill/test/framework/TestDriver.java]:
```suggestion
                  LOG.info("UNCATEGORIZED: "+(ii-DrillTestDefaults.DRILL_EXCEPTION.VALIDATION_ERROR_INVALID_SCHEMA.values().length+1)+" ("+executionFailureExceptions.get(ii).size()+")");
```

File: framework/src/main/java/org/apache/drill/test/framework/TestDriver.java [hunk 4]
```diff
@@ -448,16 +452,17 @@ else if(executionFailureExceptions.get(index)==null){
...
                 else
-                  LOG.info("\nUNCATEGORIZED: "+(ii-DrillTestDefaults.DRILL_EXCEPTION.VALIDATION_ERROR_INVALID_SCHEMA.values().length+1)+" count("+executionFailureExceptions.get(ii).size()+")");
+                  LOG.info("\nUNCATEGORIZED: "+(ii-DrillTestDefaults.DRILL_EXCEPTION.VALIDATION_ERROR_INVALID_SCHEMA.values().length+1)+"("+executionFailureExceptions.get(ii).size()+")");
```
Comment (Agirish) [inline framework/src/main/java/org/apache/drill/test/framework/TestDriver.java]:
```suggestion
                  LOG.info("\nUNCATEGORIZED: "+(ii-DrillTestDefaults.DRILL_EXCEPTION.VALIDATION_ERROR_INVALID_SCHEMA.values().length+1)+" ("+executionFailureExceptions.get(ii).size()+")");
```

**Comment by kedarbcs16** [comment]
Done

**Pull request merged**